src/index.js

```diff
@@ -236,8 +236,6 @@ function makeKCDWorkshopApp({
   handleLocationChange()
 }
 
-export default makeKCDWorkshopApp
```

Comment:
I'm not sure why this change was made. Seems stylistic. I typically prefer named exports these days, but even then I prefer exports to be defined on their own rather than inlining them (which makes them harder to identify). So I'll suggest this:

```javascript
export {makeKCDWorkshopApp}
```

(Unfortunately, GitHub's not letting me make a suggestion on this line for some reason.

Reply:
just a rename because I could not find the usage, I will update it suggested code style